contracts/src/core/AlignedLayerServiceManagerStorage.sol

```diff
@@ -11,6 +11,11 @@ abstract contract AlignedLayerServiceManagerStorage {
     // KEY is keccak256(batchMerkleRoot,senderAddress)
     mapping(bytes32 => BatchState) public batchesState;
 
+    // Bitmap representing blacklisted verifiers
+    // Each verifier is blacklisted if its corresponding bit is set to 1
+    // The verifier index follows its corresponding value in the `ProvingSystemId` enum being 0 the first verifier.
+    uint256 public blacklistedVerifiers;
```

Comment:
We should discuss if a bitmap is the best way to implement this system. It is not really easy to query since there is no on-chain way of knowing which bit is related to which Proving System.

Would a map be a better choice?

Reading is free for both cases (`cast call` and not `cast send`) , and while writing may be cheaper in a bitmap since it uses less memory, the usage of this write is really exceptional and may not justify the extra cost in usability (thus auditability and thus security), of a bitmap.

Reply:
So, as discussed, we will use a bitmap and follow the SDK. In the future, we will remove the values coded directly on the SDK and read them from a config file, so this same config file will be used to enable/disable verifiers.